lib/ansible/modules/cloud/amazon/cloudfront_invalidation.py

```diff
@@ -181,7 +181,7 @@ def get_invalidation(self, distribution_id, caller_reference):
         # find all invalidations for the distribution
         try:
             paginator = self.client.get_paginator('list_invalidations')
-            invalidations = paginator.paginate(DistributionId=distribution_id).build_full_result()['InvalidationList'].get('Items', [])
+            invalidations = paginator.paginate(DistributionId=distribution_id).build_full_result().get('InvalidationList', {}).get('Items', [])
```

Comment:
LGTM!

I think initially this may have been coded without the use of a paginator, the return of which looks like:
```
>>> c.list_invalidations(DistributionId='ESL9JHB9TV8W8')
{u'InvalidationList': {u'Marker': '', u'IsTruncated': False, u'MaxItems': 100, u'Quantity': 0}, 'ResponseMetadata': {'RetryAttempts': 0, 'HTTPStatusCode': 200, 'RequestId': '8045be5a-3683-11e8-83fa-e15870c27afd', 'HTTPHeaders': {'x-amzn-requestid': '8045be5a-3683-11e8-83fa-e15870c27afd', 'date': 'Mon, 02 Apr 2018 14:38:25 GMT', 'content-length': '210', 'content-type': 'text/xml'}}}
```
With the paginator the response is an empty dict.
```
>>> paginator = c.get_paginator('list_invalidations')
>>> paginator.paginate(DistributionId='ESL9JHB9TV8W8').build_full_result()
{}
```